packages/0x.js/test/order_state_watcher_test.ts

```diff
@@ -321,6 +321,87 @@ describe('OrderStateWatcher', () => {
                         makerToken.address, maker, ZeroEx.NULL_ADDRESS, transferAmount);
                 })().catch(done);
             });
+            it('should equal remaining amount when partially cancelled and order has fees', (done: DoneCallback) => {
+                (async () => {
+                    const takerFee = ZeroEx.toBaseUnitAmount(new BigNumber(0), 18);
+                    const makerFee = ZeroEx.toBaseUnitAmount(new BigNumber(5), 18);
```

Comment:
Let's set `18` to a constant called `ZRX_DECIMALS` in the `constants.ts` and use that everywhere rather then hard-coded `18`'s.